<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #535252;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>300</width>
         <height>800</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:  #2D2727;
border-radius: 10px;</string>
       </property>
       <widget class="QPushButton" name="btn_CPU">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>240</width>
          <height>65</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    border: none;
    color: white;
    padding: 15px 32px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
}

QPushButton:hover {
    background-color: #ef5350;
}

QPushButton:pressed {
    background-color: #e53935;
}
</string>
        </property>
        <property name="text">
         <string>Процессор</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_GPU">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>240</width>
          <height>65</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    border: none;
    color: white;
    padding: 15px 32px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
}

QPushButton:hover {
    background-color: #ef5350;
}

QPushButton:pressed {
    background-color: #e53935;
}
</string>
        </property>
        <property name="text">
         <string>Видеокарта</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_RAM">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>260</y>
          <width>240</width>
          <height>65</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    border: none;
    color: white;
    padding: 15px 32px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
}

QPushButton:hover {
    background-color: #ef5350;
}

QPushButton:pressed {
    background-color: #e53935;
}
</string>
        </property>
        <property name="text">
         <string>Оперативная память</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_motherboard">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>240</width>
          <height>65</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    border: none;
    color: white;
    padding: 15px 32px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
}

QPushButton:hover {
    background-color: #ef5350;
}

QPushButton:pressed {
    background-color: #e53935;
}
</string>
        </property>
        <property name="text">
         <string>Материнская плата</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_power_unit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>340</y>
          <width>240</width>
          <height>65</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    border: none;
    color: white;
    padding: 15px 32px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
}

QPushButton:hover {
    background-color: #ef5350;
}

QPushButton:pressed {
    background-color: #e53935;
}
</string>
        </property>
        <property name="text">
         <string>Блок питания</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_cooling">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>420</y>
          <width>240</width>
          <height>65</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    border: none;
    color: white;
    padding: 15px 32px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
}

QPushButton:hover {
    background-color: #ef5350;
}

QPushButton:pressed {
    background-color: #e53935;
}
</string>
        </property>
        <property name="text">
         <string>Охлаждение</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_case">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>500</y>
          <width>240</width>
          <height>65</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    border: none;
    color: white;
    padding: 15px 32px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
}

QPushButton:hover {
    background-color: #ef5350;
}

QPushButton:pressed {
    background-color: #e53935;
}
</string>
        </property>
        <property name="text">
         <string>Корпус</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_help">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>740</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    border: none;
    color: white;
    text-align: center;
    display: inline-block;
    font-size: 10px;
    margin: 4px 2px;
    cursor: pointer;
}

QPushButton:hover {
    background-color: #ef5350;
}

QPushButton:pressed {
    background-color: #e53935;
}
</string>
        </property>
        <property name="text">
         <string>Справка</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_storage_m2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>580</y>
          <width>240</width>
          <height>65</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    border: none;
    color: white;
    padding: 15px 32px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
}

QPushButton:hover {
    background-color: #ef5350;
}

QPushButton:pressed {
    background-color: #e53935;
}
</string>
        </property>
        <property name="text">
         <string>M2 Накопители</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_storage_sata">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>660</y>
          <width>240</width>
          <height>65</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    border: none;
    color: white;
    padding: 15px 32px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
}

QPushButton:hover {
    background-color: #ef5350;
}

QPushButton:pressed {
    background-color: #e53935;
}
</string>
        </property>
        <property name="text">
         <string>SATA Накопители</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>1259</x>
         <y>10</y>
         <width>631</width>
         <height>970</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:  #2D2727;
border-radius: 10px;</string>
       </property>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>591</width>
          <height>651</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>591</width>
           <height>651</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="layout_2">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Процессор:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="layout_CPU">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>30</number>
              </property>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Видеокарта:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="layout_GPU">
              <property name="topMargin">
               <number>30</number>
              </property>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Материнская плата:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="layout_motherboard">
              <property name="topMargin">
               <number>30</number>
              </property>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;ОЗУ:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="layout_RAM">
              <property name="topMargin">
               <number>30</number>
              </property>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Блок питания:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="layout_power_unit">
              <property name="topMargin">
               <number>30</number>
              </property>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Охлаждение:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="layout_cooling">
              <property name="topMargin">
               <number>30</number>
              </property>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_7">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Корпус:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="layout_case">
              <property name="topMargin">
               <number>30</number>
              </property>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_8">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;m2 Накопители:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="layout_storage_m2">
              <property name="topMargin">
               <number>30</number>
              </property>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_9">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;SATA Накопители:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="layout_storage_SATA">
              <property name="topMargin">
               <number>30</number>
              </property>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_10">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;Цена:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>160</y>
          <width>305</width>
          <height>55</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    border: none;
    color: white;
    padding: 15px 32px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
}

QPushButton:hover {
    background-color: #ef5350;
}

QPushButton:pressed {
    background-color: #e53935;
}
</string>
        </property>
        <property name="text">
         <string>Сформировать</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>240</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;Конфигурация:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_clear">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>240</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #f44336;
    border: none;
    color: white;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 10px;
    margin: 4px 2px;
    cursor: pointer;
}

QPushButton:hover {
    background-color: #ef5350;
}

QPushButton:pressed {
    background-color: #e53935;
}
</string>
        </property>
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_price">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>95</y>
          <width>311</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>8</y>
         <width>800</width>
         <height>970</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:  #2D2727;
border-radius: 10px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>782</width>
             <height>952</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="layout"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="btn_exit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>301</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:  #D9D9D9;
border-radius: 10px;
}
QPushButton:hover {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
	text: Выход; 
 }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/izobrazhenie-PhotoRoom_png-PhotoRoom.png</normaloff>images/izobrazhenie-PhotoRoom_png-PhotoRoom.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>250</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
